src/Workspaces/Core/Portable/Shared/TestHooks/TaskExtensions.cs

```diff
@@ -46,122 +45,5 @@ static Task CompletesTrackingOperationSlow(Task task, IDisposable token)
                     TaskScheduler.Default);
             }
         }
-
-        // Following code is copied from Microsoft.VisualStudio.Threading.TplExtensions (renamed to avoid ambiguity)
```

Comment:
i can move this back.  but i didn't think it hurt to have this available in the code-style layer as just generic task helping code.